src/lib/data/data_types.ts

```diff
@@ -56,14 +56,21 @@ interface Budget extends Comments, Document {
 type UserType = 'RSO' | 'SEPC' | 'SGA_Treasurer' | 'SGA_Assistant_Treasurer';
 // User represents a single user
 interface User extends Document {
-  user_name: string;
+  id: string;
+  name: string;
   remaining_budget: number;
   total_budget: number;
   user_type: UserType;
   pending_event: number;
   planned_event: number;
   completed_event: number;
 }
+export function userIsSGA(user: User): boolean {
+  return (
+    user.user_type === 'SGA_Treasurer' ||
+    user.user_type === 'SGA_Assistant_Treasurer'
+  );
+}
```

Comment:
the data_types file should only have types for clarity; maybe create  a utils.ts file for this function